ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationContent.tsx

```diff
@@ -1,25 +1,23 @@
-import React, { CSSProperties, ReactElement, ReactNode } from 'react';
-import { Flex, Label } from '@patternfly/react-core';
+import React, { ReactElement, ReactNode } from 'react';
+import { Label } from '@patternfly/react-core';
 
 import TechPreviewLabel from 'Components/PatternFly/TechPreviewLabel';
 
 type NavigationContentVariant = 'Deprecated' | 'TechPreview';
 
-const badgeMap: Record<string, ReactElement> = {
+const badgeMap = {
     Deprecated: (
         <Label
             isCompact
-            style={
-                {
-                    '--pf-v5-c-label--BackgroundColor': 'var(--pf-v5-global--disabled-color--200)',
-                } as CSSProperties
-            }
+            style={{
+                '--pf-v5-c-label__content--Color': 'var(--pf-v5-global--Color--dark-100)',
+            }}
         >
             Deprecated
         </Label>
     ),
     TechPreview: <TechPreviewLabel />,
-}; // TODO why does tsc build fail with missing semicolon with satisfies Record<string, ReactElement>
+} satisfies Record<string, ReactElement>;
```

Comment:
This no longer fails locally, and if it works correctly in CI we can keep the comment removed.